server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/api/MailQueueView.java

```diff
@@ -42,5 +45,9 @@
 
     ManageableMailQueue.MailQueueIterator browse();
```

Comment:
`browse` duplicates `browseReactive`. I propose we get rid of browse.

Reply:
The browse method of the MailQueueView is used in the browse method of the Mailqueue which is a synchronous API. I think it does not make sense to remove it now. 
It should be done in a PR to make the Mailqueue browse reactive.